### ISSUE TITLE
Status of package

### ISSUE THREAD

**Issue by leewillis77** [Prospress/action-scheduler-custom-tables#5]
Hi; 

I've been tracking this repo (and the main action scheduler repo as well) for a while now. 

I'm currently using https://github.com/A5hleyRich/wp-background-processing in the Google Product Feed extension (https://woocommerce.com/products/google-product-feed) to carry out background processing. When it works, this works well. However - it's proven to be quite problematic and erratic for a number of customers so I'm looking for alternatives. 

I looked at action scheduler, however since the plugin can generate a large number of background jobs, I'm not comfortable with it using CPT as the storage which let me to this repo which seems perfect. 

I notice that the composer.json claims it's version 1.0.0, but the plugin header itself declares itself as 1.0.0-dev. So, I just wanted to reach out and get a view on the stability of the package - would you consider it production ready / near production ready? If not, are you still actively working on it?

If Slack is a better medium for discussion - feel free to reach out there, I'm on the a8c or woocommerce community slacks if that's useful.

**Comment by thenbrent** [comment]
> would you consider it production ready / near production ready?

It's somewhere in between. It's in beta and we're testing it on test sites then hoping to start testing its migration on a few staging copies of real, large Subscriptions sites. Then we'll finally be testing it on the live versions of those sites, and some smaller live sites.

Once that testing is done though, it's future it to be merged into Action Scheduler itself, not kept as a separate plugin. It's being developed as a "feature plugin" for ease of testing rather than because that is its long term future.

> in the Google Product Feed extension

Eventually, Action Scheduler itself should be a good substitute without you having to mess about with using AS + this plugin. But it will be quite some time (at least 6 months) before we merge this into Action Scheduler.

If you are able to test both AS and these custom tables for your use case on test sites in the meantime, that would be a great help and it will mean the final implementation will address any feedback or issues you may have. But please don't ship this plugin with any public plugin as it will migrate all the actions for other plugins (Subscriptions, Memberships etc.) to custom tables prior to us being sure that code is ready for prime-time.

> are you still actively working on it?

Yes. Definitely. As Action Scheduler will eventually be moving to use the code here, it is something we are and will continue to work on until it is merged into Action Scheduler version 2.0.

> I notice that the composer.json claims it's version 1.0.0

Sorry for that confusion. It should really be something more like 1.0.0-beta-1.

I'll also update the README next week to clarify everything I've said above to avoid confusion about this plugins role and future.

I'll close this now but feel free to reopen or open new issues for any other questions. :)

**Comment by leewillis77** [comment]
Perfect - thanks Brent!
